modules/ClinikalAPI/sql/0_1_0-to-0_2_0_upgrade.sql

```diff
@@ -235,3 +235,9 @@ VALUES ('drugs_list', 'Drugs list');
 INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
 VALUES ('drugs_list', '9911', 'Codes', NULL);
 #EndIf
+
+#IfNotRow code_types ct_key MOH_DRUGS
+INSERT INTO `code_types` (`ct_key`, `ct_id`, `ct_seq`, `ct_mod`, `ct_just`, `ct_mask`, `ct_fee`, `ct_rel`, `ct_nofs`, `ct_diag`, `ct_active`, `ct_label`, `ct_external`, `ct_claim`, `ct_proc`, `ct_term`, `ct_problem`, `ct_drug`)
```

Comment:
this needs to be in the ImportData module, where we insert records with this code type.

Reply:
sure